geomesa-kafka/geomesa-kafka-tools/geomesa-kafka-08-tools/pom.xml

```diff
@@ -126,4 +126,36 @@
         </profile>
     </profiles>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.5.0</version>
```

Comment:
i'd move the version up to the top level pom for version management in pluginManagement section